datatableview/helpers.py

```diff
@@ -47,6 +47,12 @@ def keyed_helper(helper):
     a custom callback method that simply returns the value of the ``make_boolean_checkmark()``
     helper.
 
+    'attr' is shorthand for key=lambda o: getattr(o, attr), allowing simpler syntax like 
```

Comment:
Please update this reference to `attr` to be consistent with the rest of the docs.